setup.sh

```diff
@@ -47,10 +47,16 @@ DOCKER_PID_FILE=/var/run/docker.pid
 if [ -f "$DOCKER_PID_FILE" ]; then
     echo -e "Docker is running."
 else
-    echo -e '\e[31mDocker daemon is not running'
-    echo -e '\e[31mAborting...'
-    echo -e "\e[31mPlease start docker daemon and restart again"
-    exit
+    # for Docker desktop users
+    DOCKER_PID_FILE=/var/run/docker-desktop-proxy.pid
+    if [ -f "$DOCKER_PID_FILE" ]; then  		
+        echo -e "Docker is running."   
```

Comment:
Instead of writing another condition, check for the existence of both files in one if statement.